Commands/SpawnBoss.cs

```diff
@@ -0,0 +1,102 @@
+        {
+            var em = Helper.Server.EntityManager;
+            var playerEntity = ctx.Event.SenderCharacterEntity;
+            if (!em.Exists(playerEntity))
+            {
+                ctx.Reply("Player entity not found.");
+                return;
+            }
+
+            if (bossName.ToLower() == "all")
+            {
+                var bossPrefabs = Database.GetBossPrefabs();
+                if (bossPrefabs.Count == 0)
+                {
+                    ctx.Reply("No boss prefabs found.");
+                    return;
+                }
+
+                int bossSpawnCount = 0;
+
+                foreach (var bossPrefab in bossPrefabs)
+                {
+                    SpawnBosses(ctx, playerEntity, bossPrefab.Value, quantity);
+                    bossSpawnCount++;
+                }
+
+                ctx.Reply($"Spawned <color=#ff0>{bossSpawnCount}</color> bosses next to the player.");
+            }
+            else
+            {
+                var bossPrefab = Database.GetUnitPrefabGUID(bossName);
+                if (bossPrefab.Equals(default(PrefabGUID)))
+                {
+                    ctx.Reply($"Boss with name <color=#ff0000>{bossName}</color> not found.");
+                    return;
+                }
+
+                SpawnBosses(ctx, playerEntity, bossPrefab, quantity);
+            }
+        }
+
+        private static void SpawnBosses(ChatCommandContext ctx, Entity playerEntity, PrefabGUID bossPrefab,
+            int quantity)
+        {
+            var em = Helper.Server.EntityManager;
+            var enemyName = Database.GetUnitName(bossPrefab.ToString());
+            if (string.IsNullOrEmpty(enemyName))
+            {
+                ctx.Reply("Boss name not found.");
+                return;
+            }
+
+            try
+            {
+                var playerPosition = em.GetComponentData<LocalToWorld>(playerEntity).Position;
+                var unitSpawnerSystem = Helper.Server.GetExistingSystem<UnitSpawnerUpdateSystem>();
+
+                if (unitSpawnerSystem == null)
+                {
+                    ctx.Reply("Unit spawner system not found.");
+                    return;
+                }
+
+                for (int i = 0; i < quantity; i++)
+                {
+                    float3 spawnPosition = GetRandomSpawnPosition(playerPosition);
+                    unitSpawnerSystem.SpawnUnit(Entity.Null, bossPrefab, spawnPosition, 1, 1, 0f, 0f);
+                }
+                string spawnMessage = quantity == 1 ? $"Spawned <color=#ff0>{enemyName}</color> next to the player." :
+                    $"Spawned <color=#ff0>{quantity} {enemyName}</color> next to the player.";
+
+                ctx.Reply(spawnMessage);
+            }
+            catch (Exception e)
+            {
+                ctx.Reply($"An error occurred: {e.Message}");
+            }

```

Comment:
Don't serialize a catch all back to the user. Should be protected by the framework for this level handling. If there's a specific exception you should catch it and use `throw ctx.Error("explanation to the user")` if it's something you *should* explain to them.

Reply:
I did that because i don't have acces to the console so there is no way for me to see if there is an error ( it only show "an internal error occured" ),  so i did it that way but yes, i forgot to remove it in the end.